### ISSUE TITLE
Colorize curvature (APSS) filter produces unreadable ply file

### ISSUE THREAD

**Issue by Ainaz99** [cnr-isti-vclab/meshlab#859]
Hi,
I'm trying to apply `'Colorize curvature (APSS)'` filter on an obj file. There's no error when applying the filter, but the produced ply file is unreadable using plyfile, meshlab or blender.

```
ms = pymeshlab.MeshSet()
ms.load_new_mesh('mesh.obj')
ms.apply_filter('colorize_curvature_apss', filterscale=3, maxprojectioniters=35, curvaturetype='K1')
ms.save_current_mesh('mesh_k1.ply')
plyfile.PlyData.read('mesh_k1.ply')
```
Here's the error I get from `PlyData.read()`.

```
Traceback (most recent call last):
  File "/root/miniconda3/lib/python3.8/site-packages/plyfile.py", line 933, in _read_bin
    n = _read_array(stream, _np.dtype(len_t), 1)[0]
IndexError: index 0 is out of bounds for axis 0 with size 0

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/miniconda3/lib/python3.8/site-packages/plyfile.py", line 736, in _read_bin
    prop._read_bin(stream, byte_order)
  File "/root/miniconda3/lib/python3.8/site-packages/plyfile.py", line 935, in _read_bin
    raise StopIteration
StopIteration

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/root/miniconda3/lib/python3.8/site-packages/plyfile.py", line 394, in read
    elt._read(stream, data.text, data.byte_order)
  File "/root/miniconda3/lib/python3.8/site-packages/plyfile.py", line 656, in _read
    self._read_bin(stream, byte_order)
  File "/root/miniconda3/lib/python3.8/site-packages/plyfile.py", line 738, in _read_bin
    raise PlyElementParseError("early end-of-file",
plyfile.PlyElementParseError: element 'face': row 194340: property 'vertex_indices': early end-of-file

```
It is also unreadable in meshlab or blender, I attached the photos.
![blender](https://user-images.githubusercontent.com/69212889/101638509-682e0b80-3a43-11eb-88df-30642422cee7.png)
![meshlab](https://user-images.githubusercontent.com/69212889/101638540-70864680-3a43-11eb-9574-0b1f9ce55513.png)

And this is the link to `mesh.obj` that I'm using: https://drive.google.com/file/d/1OV2R3NtS07XyLT2KivRR3R-GHh_STWdT/view?usp=sharing

Thanks in advance for your response.

**Comment by alemuntoni** [comment]
@Ainaz99, as a first workaround, save the ply file without binary encoding.
On pymeshlab:
```
ms.save_current_mesh('mesh_k1.ply', binary=False)
```

**Comment by Ainaz99** [comment]
I also tried without binary encoding. The produced ply file was ok, but all the curvature values were zero.

**Comment by alemuntoni** [comment]
Ok. I am investigating and I found that the issue is related to the double precision version of meshlab (which is default in pymeshlab). It seems that the quality is stored as float also when declared as double in the header of the ply file.

**Comment by alemuntoni** [comment]
Solved with #860. The fix will be available in the next meshlab and pymeshlab versions!
